docs/asciidoc/static/breaking-changes.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[breaking-changes]]
+== Breaking changes
+
+Version 2.0 of Logstash has some changes that are incompatible with previous versions of Logstash. This section discusses 
+what you need to be aware of when migrating to this version.
+
+[float]
+=== Elasticsearch Output Default
+
+Starting with the 2.0 release of Logstash, the default Logstash output for Elasticsearch is HTTP. To use the `node` or
+`transport` protocols, download the {logstash}plugins-outputs-elasticsearch_java.html[Elasticsearch Java plugin]. The 
```

Comment:
can we add instructions on how to install it? See details here: https://www.elastic.co/blog/logstash-2-0-0-beta1-released "Installing java clients for Elasticsearch 1.x:"